src/mcp/tools/crashlytics/list_top_issues.ts

```diff
@@ -0,0 +1,39 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { mcpError, toContent } from "../../util.js";
+import { listTopIssues } from "../../../crashlytics/listTopIssues.js";
+
+export const list_top_issues = tool(
+    {
+        name:"list_top_issues",
+        description: "List the top issues happening in the application.",
+        inputSchema: z.object({
+            platform: z.string(),
+            packageName: z.string(),
+            issueCount: z.string().optional(),
```

Comment:
What is `issueCount`, and why is it a string and not a number? Seems likely this should be a `z.number()` and cast to string later when passed to API.

Reply:
Made it a number. Initially had it a string, but validated that we could make it a number.